plugins/woocommerce-blocks/assets/js/blocks/add-to-cart-with-options/variation-selector/edit.tsx

```diff
@@ -0,0 +1,76 @@
+/**
+ * External dependencies
+ */
+import { useBlockProps } from '@wordpress/block-editor';
+import { BlockEditProps } from '@wordpress/blocks';
+import { useProductDataContext } from '@woocommerce/shared-context';
+import {
+	// @ts-expect-error Using experimental features
+	// eslint-disable-next-line @wordpress/no-unsafe-wp-apis
+	__experimentalHeading as Heading,
+	SelectControl,
+} from '@wordpress/components';
+
+interface Attributes {
+	className?: string;
+}
+
+const AddToCartWithOptionsVariationSelectorEdit = (
+	props: BlockEditProps< Attributes >
+) => {
+	const { className } = props.attributes;
+	const { product } = useProductDataContext();
+
+	const blockProps = useBlockProps( {
+		className,
+	} );
+
+	if ( product.type !== 'variable' ) {
+		return null;
+	}
+
+	const renderAttributeSelectors = () => {
+		return Object.entries( product.attributes ).map(
+			( [ attributeKey, attribute ] ) => (
+				<div
+					className="wc-block-components-variation-selector__wrapper"
```

Comment:
I think the CSS classes in this file should have the prefix `wc-block-` instead of `wc-blocks-components-`. My understanding of the [guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/plugins/woocommerce-blocks/docs/contributors/coding-guidelines.md#naming) is that `wc-blocks-components-` should only be used by elements that might be shared across blocks (ie: some specific buttons which are shared between Mini-Cart, Cart and Checkout blocks).

Reply:
Done in [e1b698f](https://github.com/woocommerce/woocommerce/pull/53945/commits/e1b698fa4d919147a6074cb7e2325fb8dda65272)!